daemon/logger/loginfo.go

```diff
@@ -127,3 +129,45 @@ func (info *Info) ImageFullID() string {
 func (info *Info) ImageName() string {
 	return info.ContainerImageName
 }
+
+// InfoMap returns information about a container as a map.
+func (info *Info) InfoMap(keyMod func(string) string) map[string]string {
```

Comment:
We can always generalize this later, I'd rather just add what's being requested to fluentd and we can look at generalization if needed separately.